sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <map>
+
+#include "common/crypt.hpp"
```

Comment:
Have you formatted your source code with clang-format? I think this header will be moved to Line 6 after formatting.

If you don't do that, everybody that's going to edit your source code will end up having lots of changes, because their IDE will do so.